bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -179,3 +179,42 @@ def _setattr(self, name, val):
     @property
     def __version__(self):
         return self.gROOT.GetVersion()
+
+    # Overload VecOps namespace
+    @property
+    def VecOps(self):
+        if not 'VecOps' in self.__dict__:
```

Comment:
Do we need this check? Doesn't the replacement of the property by the namespace in `__dict__` already prevent that we run this code twice?

Reply:
Unfortunately no, properties are treated differently. Try the following reproducer:

```python
class Foo:
    @property
    def Bar(self):
        print('foo')
        self.__dict__['Bar'] = 42
        return 42

x = Foo()
print(x.Bar)
print(x.Bar)
```

This prints

```
foo
42
foo
42
```

so the property is not replaced by the `__dict__`.

Reply:
Ah it makes sense. In the dictionary should be the property function and not the plain function. So it makes sense it does not work.

Reply:
So we could add a plain property function via `property(func)` to the module and then replace the entry. But well... I prefer this solution here.